### PR TITLE
ISSD-757 [PRM] Freemark syntax error message appears after saving the MDF as draft

### PR DESCRIPTION
**Pull request by Manda-vit**

### CODE REVIEW

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/cancel-button-mdf-claim/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [#assign mdfClaimId = (request.getAttribute("INFO_ITEM").objectEntryId)! /]
 
 [#if mdfClaimId?has_content]
-	[#assign mdfClaim = (restClient.get("/c/mdfclaims/${mdfClaimId})!") /]
+[#assign mdfClaim = (restClient.get("/c/mdfclaims/${mdfClaimId}")!) /]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/cancel-button-mdf-claim/index.html:4]:
```suggestion
[#assign mdfClaim = (restClient.get("/c/mdfclaims/${mdfClaimId}))! /]
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by jairmedeiros** [comment]
ci:forward:force

**Pull request closed without merging**